drivers/hmis/lib/form_data/default/fragments/disability.json

```diff
@@ -671,8 +671,152 @@
       ]
     },
     {
+      "type": "GROUP",
+      "link_id": "hopwa_disability",
       "text": "HOPWA Questions",
-      "fragment": "#w4_w6_hopwa_disability"
```

Comment:
probably should remove drivers/hmis/lib/form_data/default/fragments/w4_w6_hopwa_disability.json right?